<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memory</class>
 <widget class="QWidget" name="memory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>memory</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>281</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="SFR">
    <attribute name="title">
     <string>SFR</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="Sfr_tableWidget">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>17</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>18</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>19</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>20</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>21</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>SFR</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Acc</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>PSW</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>SP</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>DPH</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>DPL</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>P0</string>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>P1</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>P2</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>P3</string>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>IP</string>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string>IE</string>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string>TMOD</string>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string>TCON</string>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string>TH0</string>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string>TL0</string>
        </property>
       </item>
       <item row="16" column="0">
        <property name="text">
         <string>TH1</string>
        </property>
       </item>
       <item row="17" column="0">
        <property name="text">
         <string>TL1</string>
        </property>
       </item>
       <item row="18" column="0">
        <property name="text">
         <string>SCON</string>
        </property>
       </item>
       <item row="19" column="0">
        <property name="text">
         <string>SBUF</string>
        </property>
       </item>
       <item row="20" column="0">
        <property name="text">
         <string>PCON</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RAM">
    <attribute name="title">
     <string>RAM</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="Ram_tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>55</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>14</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>16</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>17</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>18</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>19</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>20</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>21</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>22</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>23</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>24</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>25</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>26</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>27</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>28</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>29</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>30</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>31</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>32</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>address</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>7</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>0x00</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>0x08</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>0x10</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>0x18</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>0x20</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>0x28</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>0x30</string>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>0x38</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>0x40</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>0x48</string>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>0x50</string>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string>0x58</string>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string>0x60</string>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string>0x68</string>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string>0x70</string>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string>0x78</string>
        </property>
       </item>
       <item row="16" column="0">
        <property name="text">
         <string>0x80</string>
        </property>
       </item>
       <item row="17" column="0">
        <property name="text">
         <string>0x88</string>
        </property>
       </item>
       <item row="18" column="0">
        <property name="text">
         <string>0x90</string>
        </property>
       </item>
       <item row="19" column="0">
        <property name="text">
         <string>0x98</string>
        </property>
       </item>
       <item row="20" column="0">
        <property name="text">
         <string>0xA0</string>
        </property>
       </item>
       <item row="21" column="0">
        <property name="text">
         <string>0xA8</string>
        </property>
       </item>
       <item row="22" column="0">
        <property name="text">
         <string>0xB0</string>
        </property>
       </item>
       <item row="23" column="0">
        <property name="text">
         <string>0xB8</string>
        </property>
       </item>
       <item row="24" column="0">
        <property name="text">
         <string>0xC0</string>
        </property>
       </item>
       <item row="25" column="0">
        <property name="text">
         <string>0xC8</string>
        </property>
       </item>
       <item row="26" column="0">
        <property name="text">
         <string>0xD0</string>
        </property>
       </item>
       <item row="27" column="0">
        <property name="text">
         <string>0xD8</string>
        </property>
       </item>
       <item row="28" column="0">
        <property name="text">
         <string>0xE0</string>
        </property>
       </item>
       <item row="29" column="0">
        <property name="text">
         <string>0xE8</string>
        </property>
       </item>
       <item row="30" column="0">
        <property name="text">
         <string>0xF0</string>
        </property>
       </item>
       <item row="31" column="0">
        <property name="text">
         <string>0xF8</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ROM">
    <attribute name="title">
     <string>ROM</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
